modular_skyrat/code/datums/speech_mod/impediments.dm

```diff
@@ -0,0 +1,30 @@
+/datum/speech_mod/impediment_rl
+	soundtext = "mispronouncing \"r\" as \"l\""
+
+/datum/speech_mod/impediment_rl/handle_speech(datum/source, list/speech_args)
+	speech_args[SPEECH_MESSAGE] = replacetext(speech_args[SPEECH_MESSAGE], "r", "L")
+	speech_args[SPEECH_MESSAGE] = replacetext(speech_args[SPEECH_MESSAGE], "R", "l")
+
+/datum/speech_mod/impediment_lw
+	soundtext = "mispronouncing \"l\" as \"w\""
+
+/datum/speech_mod/impediment_rl/handle_speech(datum/source, list/speech_args)
```

Comment:
Should be `impediment_rw`
This is causing the R-to-L quirk to be replaced with the L-to-W quirk, and probably also breaks the L-to-W quirk.